### ISSUE TITLE
p2p-video-stream bug on FireFox

### ISSUE THREAD

**Issue by Suchismit4** [ScarletsFiction/SFMediaStream#23]
It's not working on firefox, and there are a lot of errors.

Can you please tell me how to stream video via the nodejs server?

edit: how to use this lib to stream the video across

**Comment by StefansArya** [comment]
Hi! Thanks for your interest for trying this lib.
But sorry, I haven't find the solution for the bug yet. It sometime works after waiting for few seconds after the stream begin, maybe it was a timing issue 💭 

To stream with video, it's similar with the [example](https://github.com/ScarletsFiction/SFMediaStream/tree/v1/example/p2p-video-stream) like for the audio. You just need to enable the `video` options for [ScarletsMediaPresenter](https://github.com/ScarletsFiction/SFMediaStream#example). and use [ScarletsVideoStreamer ](https://github.com/ScarletsFiction/SFMediaStream#scarletsvideostreamer) instead of `ScarletsAudioStreamer`.

```html
<video id="target"></video>
```
```js
let videoElement = document.querySelector('video#target');
let ... = new ScarletsVideoStreamer(videoElement, 1000);
```

By the way, if it's about peer to peer live streaming, I recommend you to use WebRTC instead. There are many [demo and example](https://webrtc.github.io/samples/) online, and actually I want to improve this lib and use WebRTC on v2 but currently I'm still have some other project to be done.

**Comment by Suchismit4** [comment]
Yeah, the problem is, WebRTC (peerjs) can't handle many users, any recommendation on that? @StefansArya
